### DISCUSSION TITLE
How to make worflow synchronous with the caller?

### DISCUSSION THREAD

**Discussion by pratprad** [Netflix/conductor#2478] [Q&A]
I want to invoke workflow and wait for it to complete to give synchronous callback.
Is that possible?

**Comment by kishorebanala** [comment]
@pratprad Conductor internally uses Task queues and worker / poller mechanism to execute tasks* (Some of the System tasks like decision, fork etc. are executed synchronously though, I'm afraid it wouldn't be possible to make workflow execution synchronous without major structural changes.
